src/main/features/response/form/models/qualifiedStatementOfTruth.ts

```diff
@@ -9,6 +9,8 @@ export class ValidationErrors {
   static readonly SIGNER_NAME_TOO_LONG: string = 'You’ve entered too many characters'
   static readonly SIGNER_ROLE_REQUIRED: string = 'Enter the role of the person signing the statement'
   static readonly SIGNER_ROLE_TOO_LONG: string = 'You’ve entered too many characters'
+  static readonly DIRECTION_QUESTIONNAIRE_REQUIRED_MESSAGE: string = 'The hearing requirement details on this page are true to the best of my knowledge.'
```

Comment:
Can you add 'Please select' to the beginning of this error?

Reply:
Done.